pkg/apis/okctl.io/v1alpha1/cluster_v1alpha1.go

```diff
@@ -196,6 +196,11 @@ type ClusterIntegrations struct {
 	// +optional
 	Autoscaler bool `json:"autoscaler,omitempty"`
 
+	// Blockstorage if set to true will install the EBS CSI block storage driver into the
+	// cluster, which makes it possible to create PersistentVolumeClaims in AWS
+	// +optional
```

Comment:
What does +optional actually do?

Reply:
It is based on k8s api machinery (if I remember correctly), essentially we could use it to autogenerate JSON schema, validation, etc.